tools/integration_tests/list_large_dir/testdata/delete_objects.sh

```diff
@@ -12,11 +13,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 set -e
-# Here $1 refers to the testBucket argument
-gcloud storage rm -r --log-http --verbosity=debug gs://$1/**
-
-# If bucket is empty it will throw an CommandException.
-if [ $? -eq 1 ]; then
-  echo "Bucket is already empty."
-  exit 0
+if [ -z "$1" ]; then
+  echo "Usage: $0 <path>"
+  exit 1
 fi
+
+PATH=$1
+# Run the gcloud command and suppress known errors
```

Comment:
Why aren't we use static bucket?

Reply:
his is a temporary workaround, a quick fix, until we implement a more permanent solution. Using a static bucket would require significant changes, potentially impacting our CD pipeline. So, for now, I've implemented this quick fix to prevent further issues while we work on a more comprehensive solution.